config/locales/en/steps.yml

```diff
@@ -184,6 +184,14 @@ en:
           page_title: Upload supporting evidence
           heading: Upload supporting evidence
           upload_file_button: Upload file
+          upload_files_heading: Upload files
+          upload_files_info: The maximum file size is 20MB. Files must be a DOC, DOCX, RTF, ODT, JPG, BMP, PNG, TIF or PDF.
+          choose_files_button: Choose files
+          uploaded_files_heading: Uploaded files
+          uploaded_files_info: When you submit this application, the caseworker will also get these files.
+          file_name: File name
+          uploaded: Uploaded
```

Comment:
uploaded and uploading aren't used as far as I can tell

We could do with a translation for uploaded as it is hardcoded in `_uploaded_file.html.erb`

So:
```
file_name: File name
uploaded_file:
    uploaded: Uploaded
```